src/ModalBase/ModalLayer.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/*  TODO: The first of these two disabled linters is okay.
```

Comment:
Maybe @wittjeff can evaluate if this second disabled linter rule is OK?

Reply:
Yea, I've asked him to take a look at the deploy previews, we definitely need aria-roles and maybe aria-labelledby for these things.